README.md

```diff
@@ -37,7 +37,7 @@ $ go test ./... -v
 
 sebak can make keypair, 'secret seed' and 'public address'.
 ```
-$ sebak key generate
+$ sebak key generate --format=default
```

Comment:
we don't need to option ' oneline'?? 
upon your code, it is state that we can use.